### PR TITLE
Add support for github enterprise APIs

### PR DESCRIPTION
**Pull request by kpdecker**

### CODE REVIEW

**Comment by aendrew** [comment]
Oh cool! LGTM, mind adding a line about this to README.md and then squashing the two commits? Thanks!

**Comment by aendrew** [comment]
Duplicate of #115. First PR to add documentation gets merged. Thanks!

**Comment by kpdecker** [comment]
@aendrew docs added.

**Comment by aendrew** [comment]
Boom! Merging now. Thanks, @kpdecker!

File: README.md [hunk 1]
```diff
@@ -50,6 +50,15 @@ See these pages for more info:
 
 [Github API OAuth Overview] (http://developer.github.com/v3/oauth)
 
+Enterprise Github instances may be specified using the `apiUrl` option:
```
Comment (nf071590) [inline README.md]: Hey @aendrew - I was wondering where this part went inside the README on master?

**Pull request merged**